WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -120,6 +131,7 @@ private extension POSItem {
         case .variableParentProduct:
             return nil
         case .coupon:
+            debugPrint("Not implemented. TODO: Make POSCoupon POSOrderable")
```

Comment:
Note: [May not need to be POSOrderable](https://github.com/woocommerce/woocommerce-ios/pull/15407), since for coupons it's enough to pass code to the API.